### PR TITLE
add all china city

### PR DESCRIPTION
**Pull request by jeffery9**
Description of the issue/feature this PR addresses:

add all china city data

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by mart-e** [comment]
Hi,

Please check #24477 we are looking to implement a per-county data modules.
Where does this list comes from? Is it something required legally? Isn't it a bit overkill to add 3000 cities?

**Comment by jeffery9** [comment]
yes. those data export from standards.

**Comment by mart-e** [comment]
Sorry but I don't understand your answer.
Why is it good to have all the cities encoded?
It makes a lot of noise for the other countries so if not required, we should avoid.

**Comment by jeffery9** [comment]
I think this should be as option for those enable city extension(base_address_city)

发送自 Windows 10 版邮件<https://go.microsoft.com/fwlink/?LinkId=550986>应用

________________________________
发件人: Martin Trigaux <notifications@github.com>
发送时间: Friday, August 17, 2018 2:49:34 PM
收件人: odoo/odoo
抄送: Jeffery CHEN Fan; Author
主题: Re: [odoo/odoo] add all china city (#23879)


Sorry but I don't understand your answer.
Why is it good to have all the cities encoded?
It makes a lot of noise for the other countries so if not required, we should avoid.

―

**Comment by pedrobaeza** [comment]
Uhm, I see this very bloated to be on base `l10n_cn`. @elicoidal what do you think about this?

**Comment by mart-e** [comment]
Or split it in a `l10n_cn_city` module?

**Comment by jeffery9** [comment]
This should be the best choice. thanks

**Comment by mart-e** [comment]
@jeffery9 could you make a PR moving the data to a new module? Thanks

**Comment by elicoidal** [comment]
@pedrobaeza thanks for the ping.
Indeed I would really prefer it in a separate module.

**Comment by Yenthe666** [comment]
@mart-e in before we get this for every single country in the world. ;)
